crates/sui-indexer-alt-framework/src/ingestion/remote_client.rs

```diff
@@ -49,15 +58,23 @@ impl IngestionClientTrait for RemoteIngestionClient {
             .join(&format!("/{checkpoint}.chk"))
             .expect("Unexpected invalid URL");
 
-        let response = self
-            .client
-            .get(url)
-            .send()
-            .await
-            .map_err(|e| FetchError::Transient {
-                reason: "request",
-                error: e.into(),
-            })?;
+        let response = with_slow_future_monitor(
```

Comment:
Is it worth moving this up to the overall ingestion client, i.e. covering fetching from any ingestion client trait implementation?